### PR TITLE
Prompts for LLMs page changes.

### PR DESCRIPTION
**Pull request by katjacksonWB**
## Description

Adds additional content to the Prompts for LLMs page to provide context about the product and information about Weave.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:  [DOCS-792](https://wandb.atlassian.net/browse/DOCS-792)

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch before submitting this PR.

[DOCS-792]: https://wandb.atlassian.net/browse/DOCS-792?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/guides/prompts/intro.md [hunk 1]
```diff
@@ -5,68 +5,105 @@ displayed_sidebar: default
...
+- Monitor and observe LLMs to better understand and evaluate performance, usage, and budgets.
 
-<!-- ## Prompts Product Suite
+## Products
 
-[Trace](#Trace) is the first of our Prompts tools -->
+### Traces
 
-## How it works
+W&B’s legacy LLM tool is called *Traces*. **Traces** allow you to track and visualize the inputs and outputs, execution flow, model architecture, and any intermediate results of your LLM chains.
```
Comment (staceysv) [inline docs/guides/prompts/intro.md]: question: why "legacy"? what are we replacing it with, "Prompts"? can we remove this word...?

File: docs/guides/prompts/intro.md [hunk 2]
```diff
@@ -5,68 +5,105 @@ displayed_sidebar: default
...
 
-W&B currently supports a tool called _Trace_. **Trace** allows you to track and visualize the inputs and outputs, execution flow, model architecture, and any intermediate results of your LLM chains. 
+Use Traces for LLM chaining, plug-in or pipelining use cases. You can use your own LLM chaining implementation or use a W&B integration provided by LLM libraries such as LangChain.
```
Comment (staceysv) [inline docs/guides/prompts/intro.md:23]: mmm technically you can just use LangChain, you do not need to use the "W&B integration" of LangChain for Traces I'm pretty sure. Do we even have a W&B integration of LangChain? Edit: yes we do, ok, news to me!

File: docs/guides/prompts/intro.md [hunk 3]
```diff
@@ -5,68 +5,105 @@ displayed_sidebar: default
...
+Use Weave to view your dataframe in your notebook with only a few lines of code:
+
+1. First, install or update to the latest version of Weave with pip:
+
+```bash
+pip install weave --upgrade
+```
+
+1. Load your dataframe into your notebook.
+2. View your dataframe with `weave.show`.
```
Comment (staceysv) [inline docs/guides/prompts/intro.md]: minor: why not weave.show()? or weave.show(dataframe)?

File: docs/guides/prompts/intro.md [hunk 4]
```diff
@@ -5,68 +5,105 @@ displayed_sidebar: default
...
+- [OpenAI GPT-3.5 Fine-Tuning](https://docs.wandb.ai/guides/integrations/openai)
+- [Hugging Face Transformers](https://docs.wandb.ai/guides/integrations/huggingface)
+
+## Getting Started
+
+We recommend you go through the Prompts [Quickstart](https://docs.wandb.ai/guides/prompts/quickstart) guide, which will walk you through logging a custom LLM pipeline with Trace. A [colab](http://wandb.me/prompts-quickstart) version of the guide is also available. 
+
+## Next Steps
 
+- Check out more detailed documentation on Weave, Trace, or our [OpenAI](https://docs.wandb.ai/guides/prompts/openai) Integration.
```
Comment (staceysv) [inline docs/guides/prompts/intro.md]: minor: if we're going to provide a link for OpenAI, can we also link "Weave" to the Weave github repo? and not sure where Traces could go, but maybe the traces examples in the repo...?

**Review by staceysv — APPROVED** [review]
Looks great to me, thank you! Left some minor comments, I don't think any are blocking.

File: docs/guides/prompts/intro.md (hunk 4 above)
Comment (katjacksonWB) [reply docs/guides/prompts/intro.md]: Good point. I forgot the links, so I'll add those.

**Pull request merged**